### ISSUE TITLE
Uninstall won't delete folder /ProgramData/ums_dir/

### ISSUE THREAD

**Issue by pipin77** [UniversalMediaServer/UniversalMediaServer#1520]
After uninstalling UMS the ums folder in /ProgramData is still there. Files in there and subdirectories too.

**Comment by Nadahar** [comment]
The configuration folder won't be deleted when uninstalling, as is normal for most software. That way you keep your configuration if you install it again later.

Your other problems seems the be because you have run the uninstaller while UMS was still running. That's not how it's intended to be done, so I guess quite a few things are left as they will have been locked during uninstallation.

**Comment by pipin77** [comment]
Yeah, I had the problem after updating to 7.1.0 that UMS did not start completely to the GUI. Only a new install after manually deleting all folders worked.
